### ISSUE TITLE
The validation set loss and score of your model with LB 0.71?

### ISSUE THREAD

**Issue by hh-xiaohu** [tubo213/kaggle-child-mind-institute-detect-sleep-states#55]
May I ask for the validation set loss and score of your model with LB 0.71?  I trained a model following your configuration. The loss steadily decreased and converged, but the score showed fluctuations around 0.002."

**Comment by tubo213** [comment]
@hh-xiaohu 
Here are the results of the LB0.714 experiment
https://api.wandb.ai/links/213tubo/ub5qcxbh

Please comment on [Disscussion](https://www.kaggle.com/competitions/child-mind-institute-detect-sleep-states/discussion/452940#2530545) for a summary, as exchanges on github may fall under the category of private sharing.
